### ISSUE TITLE
simplify names when running exporter.phylogenetics

### ISSUE THREAD

**Issue by nataliacp** [Anaphory/lexedata#119]
It would be good to strip diacritics, spaces, brackets and stuff like that from language names when exporting matrices. many phylogenetic programs have problems with those.

**Comment by Anaphory** [comment]
True, that's why I prefer to use IDs instead of names and why I prefer my IDs to not have that noise.
So, how about

 - A tool to strip special characters from language IDs (we would have loved that tool yesterday) and
 - phylogenetics export defaulting to using IDs, and names only on request?
